.golangci.yaml

```diff
@@ -11,6 +11,7 @@ run:
   - ^upi
   go: '1.19'
```

Comment:
Maybe we can bump this to 1.20? We couldn't before because of the linter version.

Reply:
done!